ibm/data_source_ibm_cloudant.go

```diff
@@ -0,0 +1,136 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceIBMCloudant() *schema.Resource {
+	riSchema := dataSourceIBMResourceInstance().Schema
+
+	riSchema["service"] = &schema.Schema{
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "The service type of the instance",
+	}
+
+	riSchema["version"] = &schema.Schema{
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Vendor version.",
+	}
+
+	riSchema["features"] = &schema.Schema{
+		Type:        schema.TypeList,
+		Computed:    true,
+		Description: "List of enabled optional features.",
+		Elem: &schema.Schema{
+			Type: schema.TypeString,
+		},
+	}
+
+	riSchema["features_flags"] = &schema.Schema{
+		Type:        schema.TypeList,
+		Computed:    true,
+		Description: "List of feature flags.",
+		Elem: &schema.Schema{
+			Type: schema.TypeString,
+		},
+	}
+
+	riSchema["include_data_events"] = &schema.Schema{
```

Comment:
All the IBM provided resource and data starts with `ibm_` prefix and also contains the service name that it relates like: `ibm_cos` [doc-ref](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/cos_bucket)

The generator produced the following prefix `ibm_cloudant`. I am okay with this prefix.
```suggestion
	riSchema["ibm_cloudant_include_data_events"] = &schema.Schema{
```

Reply:
How you can define this in a `resource` statement? This can be configured only with service setup?

Reply:
Our resource and data source start with `ibm_` prefix, they named `ibm_cloudant`. You are confusing a name of resource and names of attributes within resource here, take a look at any other module withing project. The same is true for generated modules as well, for example in generated module  `data_source_ibm_cloudant_activity_tracker_events.go` schema attribute called "types", not "ibm_cloudant_activity_tracker_events_types"

Reply:
> How you can define this in a resource statement? This can be configured only with service setup?

Like this:

```hcl
resource "ibm_cloudant" "terraform-test" {
  name     = "terraform-test"
  plan     = "standard"
  location = "us-south"

  include_data_events = true  
}
```